ReadMe.md

```diff
@@ -0,0 +1,37 @@
+
+To test the changes, select build or compile from the Visual Studio menu. It should take a moment to compile the code into a DLL file. If there are errors or warnings, they will show up at the bottom. You will need to fix any errors before a DLL is produced. A DLL can still be produced when there are warnings, but there is a good chance the code is wrong, so warnings should be fixed anyway. If you get errors, the fastest way to fix them is usually to search the error message online. Good C++ resources or tutorials may help. Stackoverflow typically has lots of good answers. If nothing generic helps, or the problem is hard to search for, it may be best to post a question on [Outpost Universe Forums](https://forum.outpost2.net/). In particular, if you find a bug in the LevelTemplate code, or the MissionSDK, or find documentation is lacking or wrong or hard to follow, we would love to hear about it. Our efforts are largely dependent on what people ask about.
+
+Once you have a DLL file, it needs to be copied to the Outpost 2 folder. It will also need to be named appropriately, based on the game type. For colony games, the DLL filename must begin with a "c". Be sure to keep the DLL filename to 7 characters or less, plus the extension (example: "cLevel.dll" uses 6 characters for the name). If the name exceeds 7 characters, you may see duplicate entries in the games list within Outpost 2. Optionally, as part of the build, Visual Studio can be configured to set the output DLL name, and automatically copy the DLL to the Outpost 2 folder. The filename is controlled by the project's build output setting. Copying the DLL is controlled by a post-build step. The TemplateProject comes pre-configured with a post-build step to copy the DLL, provided it finds an `Outpost2Path` environment variable. As each person may store their copy of Outpost 2 in a different folder, the post build step depends on this environment variable to find it. As a benefit, if you set an environment variable once on your system, it will be usable by all level projects, without further configuration. Check Google for how to set an environment variable on Windows.
```

Comment:
This is good info, but is a lot for one paragraph. Perhaps one paragraph to describe the naming convention and one to discuss the environment variable / post build event.

What do you think about copying the table here: https://wiki.outpost2.net/doku.php?id=op2_sdk:scenario_naming_conventions. That way it is easy to know the required prefix.

Reply:
Thank you for the link. I'll try to split and update that a bit.

On the wiki, it seems the autodemo entry is not quite correct. I believe the prefix is simply "a". The rest I believe is all arbitrary.